packages/storybook/stories/va-alert.stories.jsx

```diff
@@ -60,8 +64,26 @@ const Template = ({
   closeable,
   'full-width': fullWidth,
   headline,
-  onclose,
+  vaClose,
+  storybookOnlyClose,
 }) => {
+  if (storybookOnlyClose)
```

Comment:
Alternatively, I can create a separate template for a closeable alert.

Reply:
If we have alerts that aren't closeable, that might be a good idea. Is the default not closeable?

Reply:
I updated it to display bindings if the `onVaClose` prop is present and removed the storybookOnly prop.